### PR TITLE
feat added nf-validation

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] Add nf-validation
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by jemten — COMMENTED** [review]
A couple of small comments

File: workflows/tomte.nf [hunk 1]
```diff
@@ -103,13 +102,48 @@ workflow TOMTE {
...
+    ch_case_info = ch_samples.toList().map { create_case_channel(it) }
+
+    //if (params.input) {
+    //    ch_input = Channel.fromPath(params.input)
+    //    CHECK_INPUT (ch_input)
+    //    ch_versions = ch_versions.mix(CHECK_INPUT.out.versions)
+    //} else {
+    //    exit 1, 'Input samplesheet not specified!'
+    //}
+
```
[the remark this answers is not in the document]
Comment (Lucpen) [reply workflows/tomte.nf:146]: yes :)

File: workflows/tomte.nf [hunk 2]
```diff
@@ -103,13 +102,48 @@ workflow TOMTE {
...
+        .tap { ch_original_input }
+        .map { meta, fastq1, fastq2 -> meta.id }
+        .reduce([:]) { counts, sample -> //get counts of each sample in the samplesheet - for groupTuple
+            counts[sample] = (counts[sample] ?: 0) + 1
+            counts
+        }
+        .combine( ch_original_input )
+        .map { counts, meta, fastq1, fastq2 ->
+            new_meta = meta + [num_lanes:counts[meta.id],
+                        read_group:"\'@RG\\tID:"+ fastq1.toString().split('/')[-1] + "\\tPL:ILLUMINA\\tSM:"+meta.id+"\'"]
```
[the remark this answers is not in the document]
Comment (Lucpen) [reply workflows/tomte.nf]: Done

**Review by jemten — APPROVED** [review]
👍

**Pull request merged**